docs/react/guides/migrate-from-v1-to-v2.md

```diff
@@ -154,7 +154,7 @@ const { data: balance, queryKey } = useBalance({
 useEffect(() => {
   if (blockNumber % 5 === 0) // [!code focus]
     queryClient.invalidateQueries({ queryKey }) // [!code focus]
-}, [blockNumber])
+}, [blockNumber, queryKey, queryClient])
```

Comment:
`queryKey` will remain the same, but can add `queryClient`.

```suggestion
}, [blockNumber, queryClient])
```

Reply:
I dont think it hurts even though we include it, otherwise the react eslint plugin will keep showing the warning:

![CleanShot-xt0gYPAH@2x](https://github.com/wevm/wagmi/assets/126733611/e6c37c3d-2632-455d-993a-3dc6d237b1a0)

Unless we suppress with eslint comment.